tests/Issues/Fixture/coariant_mixture.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Core\Tests\Issues\Fixture;
+
+final class CoriantMixture
+{
+    private function parseArgument(string $token, array $all)
+    {
+        if (($inputArgument = $all[$key = array_key_first($all)] ?? null) && 'command' === $inputArgument->getName()) {
+        }
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Core\Tests\Issues\Fixture;
+
+final class CoriantMixture
+{
+    /**
+     * @param string $token
+     */
+    private function parseArgument($token, array $all)
+    {
+        reset($all);
+        if (($inputArgument = isset($all[$key = key($all)]) ? $all[$key = key($all)] : null) && 'command' === $inputArgument->getName()) {
+        }
+    }
```

Comment:
@samsonasik BTW, today I find one nasty bug in Rector. 
This might be one reason why the downgrade rules were not working sometimes.

This code previously failed and missed all the removed types. Causing missmatch:

![image](https://user-images.githubusercontent.com/924196/117499081-ff5ba880-af7a-11eb-9dba-476f1302665b.png)

Reply:
@TomasVotruba thank you 👍